--- protoc-gen-validate/validate/BUILD	2021-04-23 09:53:23.000000000 +1000
+++ protoc-gen-validate-copy/validate/BUILD	2021-04-23 10:05:09.000000000 +1000
@@ -1,5 +1,6 @@
 load("@com_google_protobuf//:protobuf.bzl", "py_proto_library")
 load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
 load("@rules_cc//cc:defs.bzl", "cc_library", "cc_proto_library")
 load("@rules_java//java:defs.bzl", "java_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
@@ -30,6 +31,22 @@
     deps = ["@com_google_protobuf//:protobuf_python"],
 )

+go_library(
+    name = "go_custom_library",
+    srcs = [
+        "validate.pb.go",
+    ],
+    importpath = "github.com/envoyproxy/protoc-gen-validate/validate",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@org_golang_google_protobuf//reflect/protoreflect:go_default_library",
+        "@org_golang_google_protobuf//runtime/protoimpl:go_default_library",
+        "@org_golang_google_protobuf//types/descriptorpb:go_default_library",
+        "@org_golang_google_protobuf//types/known/durationpb:go_default_library",
+        "@org_golang_google_protobuf//types/known/timestamppb:go_default_library",
+    ],
+)
+
 go_proto_library(
     name = "go_default_library",
     importpath = "github.com/envoyproxy/protoc-gen-validate/validate",